api/client/commands.go

```diff
@@ -2722,6 +2732,7 @@ func (s *containerStats) Display(w io.Writer) error {
 
 func (cli *DockerCli) CmdStats(args ...string) error {
 	cmd := cli.Subcmd("stats", "CONTAINER [CONTAINER...]", "Display a live stream of one or more containers' resource usage statistics", true)
+	noStream := cmd.Bool([]string{"-no-stream"}, false, "Disable streaming stats and only pull the first result")
```

Comment:
@tiborvass I agree it would be better, but I hate asking for contributors to learn about those (for example the fish completion requires to use a python script which generates completion file from Docker's help message) :-(